common/djangoapps/student/models.py

```diff
@@ -200,12 +200,18 @@ def anonymous_id_for_user(user, course_id, save=True):
         monitoring.increment('temp_anon_uid_v2.fetched_existing')
     else:
         # include the secret key as a salt, and to make the ids unique across different LMS installs.
-        hasher = hashlib.md5()
+        hasher = hashlib.shake_128()
```

Comment:
No test changes?

Incidentally, I'd call this a `fix` in the nomenclature of Conventional Commits.

Reply:
Oh no, yep, there will be.

Reply:
hmmm, I can kinda see, but still think it doesn't really work. We are not fixing a bug, we are just moving to a better algorithm.

Reply:
Honestly, I'm not real thrilled by the options suggested in that spec. :-) This isn't a feature, and it's not _exactly_ a bug fix, it's... an "improvement".